tests/fakeapi/aiohttp_handlers.py

```diff
@@ -78,6 +78,22 @@ async def aiohttp_token_info(token_info):
     return aiohttp.web.json_response(data=token_info)
 
 
+async def aiohttp_all_auth(token_info):
+    return await aiohttp_token_info(token_info)
+
+
+async def aiohttp_async_auth(token_info):
+    return await aiohttp_token_info(token_info)
+
+
+async def aiohttp_bearer_auth(token_info):
+    return await aiohttp_token_info(token_info)
+
+
+async def aiohttp_async_bearer_auth(token_info):
+    return await aiohttp_token_info(token_info)
+
+
```

Comment:
What is this solving?
Should all of these be async or only the ones with async in the name?

Reply:
The operationId in the OpenAPI spec needs to be unique to be valid. Connexion uses the operationId to link to the Python code that needs to be executed. Previously all endpoints in [tests/fixtures/aiohttp/openapi_secure.yaml](https://github.com/ml6team/connexion/pull/2/files#diff-6ccbde14498e093481f6333cbe9ba55610f1b70234bbd20311c0bece521a3653R11) pointed to the same Python function which led to validation errors of the spec, so I decoupled this into separate functions.